src/Umbraco.Web.UI.Client/src/less/components/umb-content-grid.less

```diff
@@ -26,7 +26,9 @@
    justify-content: center;
 }
 
-.umb-content-grid__icon {
+.umb-content-grid__icon,
+.umb-content-grid__icon[class^="icon-"],
+.umb-content-grid__icon[class*=" icon-"] {
```

Comment:
Fix issue with custom icons in grid layout.

**Before**
![image](https://cloud.githubusercontent.com/assets/2919859/14226153/d0837d1a-f8d9-11e5-9c29-222d02698ac9.png)

**After**
![image](https://cloud.githubusercontent.com/assets/2919859/14226156/df61268e-f8d9-11e5-9460-9e61fc9f3b6d.png)